lang/csharp/src/apache/main/Generic/GenericDatumReader.cs

```diff
@@ -194,13 +195,13 @@ public void AddElements( object arrayObj, int elements, int index, ReadItem item
 
             private static void SizeTo(ref object array, int targetSize)
             {
-                var o = (object[]) array;
+                var o = (object[])array;
                 Array.Resize(ref o, targetSize);
                 array = o;
             }
         }
 
-        class GenericMapAccess : MapAccess
+        private class GenericMapAccess : MapAccess
```

Comment:
By default methods are private (technically internal) unless otherwise set by an access modifier.
https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/access-modifiers
"Classes, records, and structs declared directly within a namespace (in other words, that aren't nested within other classes or structs) can be either public or internal. internal is the default if no access modifier is specified."